Builds and It's Completely Different but Also Still Build (#18955)
cloudflare/cloudflare-docs: src/content/docs/pages/configuration/git-integration/gitlab-integration.mdx

---
pcx_content_type: concept
title: GitLab integration
---

> Cloudflare supports connecting Cloudflare Pages to your GitLab repositories, and will automatically deploy your code everytime you push a change to a branch.

Reviewer: ```suggestion
You can connect each Cloudflare Pages project to a GitLab repository, and Cloudflare will automatically deploy your code every time you push a change to a branch.
```
